brave/brave-core
# Add contributions back into brave://rewards-internals page

Resolves https://github.com/brave/brave-browser/issues/8633

## Submitter Checklist:

- Verified that these changes build without errors on
- Verified that these changes pass automated tests (unit, browser, security tests) on

## Test Plan:

- Enable Rewards
- Claim grant
- Make a few contributions
- Load brave://rewards-internals
- Visit the "Contributions" tab and verify that contribution/publisher info looks correct

## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on.
